README.md

```diff
@@ -65,9 +65,18 @@ The fastest ways to run PostgreSQL with pgvectorscale are:
 
 You can install pgvectorscale from source and install it in an existing PostgreSQL server
 
+1. Ensure the pgvector extension is available:
```

Comment:
Can you move this to after the user connects to the database please. Then the process is:

Build the extension
Connect to database
Check for vector (if no, install)
Enable pgvectorscale